nix-support-dev/lainchan.nix

```diff
@@ -49,6 +49,7 @@ in
     ffmpeg
     libiconv
     phpExtensions.memcached
+    exiftool
```

Comment:
remove exiftool, libiconv, graphicsmagick, imagemagick, and gifsicle from systemPackages. I think this would warrant another couple of manual tests, but these are libraries used by the php code, and thus do not belong in systemPackages, which should be thought of as the users environment on the system. Each service has it's own environment.

They should 100% somehow live in the php path though!